summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java

```diff
@@ -82,6 +82,10 @@ public Void apply(IN v) {
     return new Service<Memory, JMemoryService<K, V>, K, V>(new JMemoryService(service));
   }
 
+<<<<<<< HEAD
+=======
+  private Memory platform = new Memory(new JobId("inMem"));
```

Comment:
No, we fixed it with [this](https://github.com/twitter/summingbird/blob/develop/summingbird-core-java/src/main/java/com/twitter/summingbird/memory/javaapi/JMemory.java#L33) in the later version. We can take the HEAD version here.